### PR TITLE
Add a blank .travis.yml for basic CI integration.

### PR DESCRIPTION
**Pull request by jsirois**
Simulated corrected description.

### CODE REVIEW

**Review by jsirois — COMMENTED** [review]
Please no here-docs, otherwise LGTM

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+sudo: false
+language: python
+script: |
```
Comment (jsirois) [inline .travis.yml]: Sorry - right here - no "here docs" here please.

Comment (jsirois) [reply .travis.yml]: Fixed

**Review by jsirois — COMMENTED** [review]
LGTM!

**Review by jsirois-test — APPROVED** [review]
LGTM!

**Pull request merged**